pkg/planner/core/casetest/instanceplancache/BUILD.bazel

```diff
@@ -12,7 +12,7 @@ go_test(
         "others_test.go",
```

Comment:
Can you add some performance data in this PR.

Reply:
Thanks, I've added a profile pic in the Description.